cmd/plugin/cli/install.go

```diff
@@ -30,6 +30,7 @@ func InstallCmd() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.Flags().Bool(installer.WaitFlag, false, "wait for operation finish")
```

Comment:
Nit: I think this should be a bit more descriptive, maybe:
 `wait for storagos cluster to enter running phase` ?